Libraries/PeriphDrivers/Source/SPIXF/spixf_me55.c

```diff
@@ -41,14 +43,66 @@
 #include "spixf.h"
 #include "spixf_reva.h"
 
+#if defined(SPIXF_RAM)
+
+#include "gpio.h"
+#include "gpio_regs.h"
+#include "gpio_reva_regs.h"
+#include "mxc_pins.h"
+
+mxc_gpio_cfg_t gpio_cfg_spixf_ram = { MXC_GPIO1, (MXC_GPIO_PIN_22 | MXC_GPIO_PIN_23 | MXC_GPIO_PIN_24 |
+                                        MXC_GPIO_PIN_25 | MXC_GPIO_PIN_26 | MXC_GPIO_PIN_27), MXC_GPIO_FUNC_ALT1,
+                                        MXC_GPIO_PAD_NONE, MXC_GPIO_VSSEL_VDDIO, MXC_GPIO_DRVSTR_0 };
```

Comment:
Is this the new official pin set for the ME55? If so, it might be a good idea to add this to `Libraries/PeriphDrivers/Source/SYS/pins_me55.c` and `Libraries/PeriphDrivers/Include/MAX32572/mxc_pins.h` instead.

Reply:
No, this is not a new pin set.  This pin set is same with the `gpio_cfg_spixf` pin set in `Libraries/PeriphDrivers/Source/SYS/pins_me55.c` file. The only difference is that the `gpio_cfg_spixf` is declared as a const. Since we should not touch the flash area when executing `SPIXF` functions, the new pin set (`gpio_cfg_spixf_ram`) should not be const. 

As a summary, this is not a new pin set, it is just a non-const version of existing pin set.

Reply:
Got it. I'd recommend leaving a comment adding this in case someone unfamiliar in the future wonders about this!